src/components/player.js

```diff
@@ -72,6 +72,24 @@ class Player extends vjsPlayer {
     return audioTrack;
   }
 
+  /**
+   * Calculates an array of ranges based on the `seekable()` data.
```

Comment:
Would be great to have a link to the `seekable` function for more context: https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/seekable